support/scripts/checkpatch.pl

```diff
@@ -48,17 +56,25 @@
 my @ignore = ();
 my $help = 0;
 my $configuration_file = ".checkpatch.conf";
-my $max_line_length = 80;
+my $max_line_length = 100;
```

Comment:
```suggestion
my $max_line_length = 80;
```
Or maybe change here?

Reply:
Actually it's not required to patch `checkpatch` for that, because it allows overriding it via a command line parameter. This series includes https://github.com/unikraft/unikraft/pull/1289/commits/45f16971a644c35cc562175702d2cc2ad907e40d to retain the 80-char limit in `checkpatch.conf`.